app/controllers/items_controller.rb

```diff
@@ -0,0 +1,26 @@
+class ItemsController < ApplicationController
+  def new
+    @item = Item.new
+    @organization = Organization.find(params[:organization_id])
+  end
+
+  def create
+    @item = Item.new(item_params)
+    @organization = Organization.find(params[:organization_id])
+    if @item.save
+      flash[:notice] = "Your item has been added."
+      redirect_to organization_path(params[:organization_id])
+    else
+      flash[:errors] = @item.errors.full_messages.join(", ")
+      render "new"
+    end
+  end
+
+  protected
+
+  def item_params
+    params.require(:item)
+    .permit(:name, :quantity, :url, :description, :status)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.